### PR TITLE
Update MobiledgeX_RuntimeTests.cs  EDGECLOUD-2811

### PR DESCRIPTION
**Pull request by ah1053**
Screen Shot 2020-07-01 at 10 36 41 AM

1. Remove VerifyLocation Test since it will always return false in Editor
2. GetURL Tests

### CODE REVIEW

File: Tests/Runtime/MobiledgeX_RuntimeTests.cs [hunk 1]
```diff
@@ -94,31 +92,32 @@ public void FindCloudlet(string orgName, string appName, string appVers)
...
         {
+            MobiledgeXIntegration.orgName = orgName;
+            MobiledgeXIntegration.appName = appName;
+            MobiledgeXIntegration.appVers = appVers;
             var task = Task.Run(async () =>
             {
-                bool registered = await RegisterHelper();
-                Assert.True(registered, "Unable to register");
-                return await VerifyLocationHelper();
+                return await GetUrlHelper(orgName, appName, appVers,"http");
```
Comment (franklin-huang-mobiledgex) [inline Tests/Runtime/MobiledgeX_RuntimeTests.cs:102]: nit: need a space between ',' and "http"

File: Tests/Runtime/MobiledgeX_RuntimeTests.cs [hunk 2]
```diff
@@ -213,32 +212,38 @@ public async Task<bool> FindCloudletHelper()
...
-
-            FindCloudletReply reply = await integration.matchingEngine.RegisterAndFindCloudlet(orgName, appName, appVers, testLocation, MatchingEngine.wifiCarrier);
+            FindCloudletReply reply = await integration.matchingEngine.RegisterAndFindCloudlet(orgName, appName, appVers, testLocation, "");
             await Task.Delay(TimeSpan.FromMilliseconds(200));
-            string uri = reply.fqdn;
-            return uri;
+            DistributedMatchEngine.AppPort appPort;
+
+            switch (proto)
+			{
```
Comment (franklin-huang-mobiledgex) [inline Tests/Runtime/MobiledgeX_RuntimeTests.cs:223]: nit weird indent

File: Tests/Runtime/MobiledgeX_RuntimeTests.cs [hunk 3]
```diff
@@ -213,32 +212,38 @@ public async Task<bool> FindCloudletHelper()
...
+                case "http":
+                     appPort = integration.GetAppPort(DistributedMatchEngine.LProto.L_PROTO_HTTP);
```
Comment (franklin-huang-mobiledgex) [inline Tests/Runtime/MobiledgeX_RuntimeTests.cs:225]: I think we use L_PROTO_TCP for this. Backend doesn't support L_PROTO_HTTP

Comment (franklin-huang-mobiledgex) [reply Tests/Runtime/MobiledgeX_RuntimeTests.cs:225]: Same for the default

File: Tests/Runtime/MobiledgeX_RuntimeTests.cs [hunk 4]
```diff
@@ -213,32 +212,38 @@ public async Task<bool> FindCloudletHelper()
...
 
         public async Task<string> WebSocketTestHelper(string orgName, string appName, string appVers)
         {
-            FindCloudletReply reply = await integration.matchingEngine.RegisterAndFindCloudlet(orgName, appName, appVers, testLocation, MatchingEngine.wifiCarrier);
-            await Task.Delay(TimeSpan.FromMilliseconds(200));
-            Dictionary<int, AppPort> appPortsDict = integration.matchingEngine.GetTCPAppPorts(reply);
-            int public_port = reply.ports[0].public_port;
-            AppPort appPort = appPortsDict[public_port];
-            ClientWebSocket ws = await integration.matchingEngine.GetWebsocketConnection(reply, appPort, public_port, 5000, "?roomid=testing&pName=MobiledgeX&pCharacter=2");
+            FindCloudletReply reply = await integration.matchingEngine.RegisterAndFindCloudlet(orgName, appName, appVers, testLocation, "");
```
Comment (franklin-huang-mobiledgex) [inline Tests/Runtime/MobiledgeX_RuntimeTests.cs:240]: carrier defaults to "" in the C# SDK, so you don't need it here

File: Tests/Runtime/MobiledgeX_RuntimeTests.cs (hunk 3 above)
Comment (ah1053) [reply Tests/Runtime/MobiledgeX_RuntimeTests.cs:222]: We had a discussion about this and  GetAppPort(http Proto ) frankling fixed so it returns TCP ports

Comment (lgarner) [reply Tests/Runtime/MobiledgeX_RuntimeTests.cs:222]: ^ TCP AppPort, same return type, right?

Comment (ah1053) [reply Tests/Runtime/MobiledgeX_RuntimeTests.cs:222]: Yes
Screen Shot 2020-07-02 at 4 21 43 PM
Screen Shot 2020-07-02 at 4 20 51 PM

**Pull request merged**